tests/src/JIT/HardwareIntrinsics/X86/Shared/ScalarUnOpTest.template

```diff
@@ -37,11 +37,17 @@ namespace JIT.HardwareIntrinsics.X86
                 // Validates passing a local works, using Unsafe.ReadUnaligned
                 test.RunLclVarScenario_UnsafeRead();
 
-                // Validates passing the field of a local works
-                test.RunLclFldScenario();
+                // Validates passing the field of a local class works
```

Comment:
Same changes made to other files were made to this one, as well as `ScalarBinOpTest` and `ScalarSimdUnOpTest`. `ScalarSimdUnOpTest` was renamed from `ScalarUnOpTest`. `ScalarBinOpTest` and the new `ScalarUnOpTest` were added in the `Bmi1`/`Bmi2` PR and needed to cover these scenarios as well.